.github/workflows/ci.yml

```diff
@@ -839,6 +839,9 @@ jobs:
             # this suite is not meant to be run with different configs
             - config: default
               suite: suite-delta-lake-oss
+            # this suite is not meant to be run with different configs
```

Comment:
Follow-up is fine. Let's update `etc` directory in trino-server-dev module so that we can test Hudi connector quickly.